### ISSUE TITLE
Silent failure for source with trailing close paren

### ISSUE THREAD

**Issue by schmave** [brython-dev/brython#946]
Put this code into the [editor](http://brython.info/tests/editor.html?lang=en):
```python
x = 5

)
```
click the "run" button, and there is no output. The JS console contains:
```
no __str__ for RangeError: Invalid count value
    at String.repeat (<anonymous>)
    at Object.int.__mul__ (py_int.js:306)
    at method (py_builtin_functions.js:908)
    at format_exception198 (eval at run_py (py_import.js:260), <anonymous>:161:113)
    at format_exc197 (eval at run_py (py_import.js:260), <anonymous>:82:109)
    at print_exc196 (eval at run_py (py_import.js:260), <anonymous>:41:133)
    at run35 (eval at run_py (py_import.js:260), <anonymous>:727:87)
    at lambda_dh4c6x36_514 (eval at $B.loop (py2js.js:9335), <anonymous>:153:107)
    at HTMLButtonElement.<anonymous> (py_dom.js:1011)
py_string.js:1707 err  Error
    at Object._b_.AttributeError.$factory (eval at $make_exc (py_exceptions.js:421), <anonymous>:59:15)
    at Object.$B.$getattr (py_builtin_functions.js:825)
    at str.$factory (py_string.js:1701)
    at write29 (eval at run_py (py_import.js:260), <anonymous>:361:54)
    at method (py_object.js:230)
    at HTMLButtonElement.<anonymous> (py_dom.js:1020)
py_string.js:1712 Warning - no method __str__ or __repr__, default to toString RangeError: Invalid count value
    at String.repeat (<anonymous>)
    at Object.int.__mul__ (py_int.js:306)
    at method (py_builtin_functions.js:908)
    at format_exception198 (eval at run_py (py_import.js:260), <anonymous>:161:113)
    at format_exc197 (eval at run_py (py_import.js:260), <anonymous>:82:109)
    at print_exc196 (eval at run_py (py_import.js:260), <anonymous>:41:133)
    at run35 (eval at run_py (py_import.js:260), <anonymous>:727:87)
    at lambda_dh4c6x36_514 (eval at $B.loop (py2js.js:9335), <anonymous>:153:107)
    at HTMLButtonElement.<anonymous> (py_dom.js:1011)
```

**Comment by PierreQuentel** [comment]
@schmave 
The bug exists at brython.info, but not with the current development version (I don't know since which commit). Can you check ?
Thanks !

**Comment by schmave** [comment]
I agree, it works fine in the latest master. Looks like it was fixed by 5b5e44980cfcfe73bc57a853ff8c0254f404195f. Thanks for the fix!
